server/api/handler/login.go

```diff
@@ -45,12 +46,24 @@ func (lh *loginHandler) Init(ctx echo.Context) error {
 		webauthn.WithUserVerification(h.config.WebauthnConfig.UserVerification),
 	)
 	if err != nil {
+		auditErr := h.auditLog.Create(ctx, h.tenant, models.AuditLogWebAuthnAuthenticationInitFailed, nil, nil)
```

Comment:
Would be good to include the error in the audit log, to have a cause why it failed. (Applies to all audit logs)